reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ReelPicker.kt

```diff
@@ -28,6 +29,8 @@ import com.innowisegroup.reelpicker.picker.ui.DatePickerFragment.Companion.WRAP_
 import com.innowisegroup.reelpicker.picker.ui.PagerAdapter
 import com.innowisegroup.reelpicker.picker.ui.TimePickerFragment
 import com.innowisegroup.reelpicker.picker.ui.TimePickerFragment.Companion.LOCAL_TIME
+import com.innowisegroup.reelpicker.picker.ui.TimePickerFragment.Companion.MAX_TIME
+import com.innowisegroup.reelpicker.picker.ui.TimePickerFragment.Companion.MIN_TIME
 
 class ReelPicker : DialogFragment() {
```

Comment:
1. onCreateViewFunction seems to be really huge. Let's separate it to improve readability.
2. Are these values really can be null. You check them for nullability before passing to ReelPicker arguments. So it cannot be null. Let's make them lateinit var and garantee to initialize in onCreateView and avoid nullability for these properties:

    private lateinit var initialLocalDateTime: LocalDateTime
    private lateinit var minLocalDateTime: LocalDateTime
    private lateinit var maxLocalDateTime: LocalDateTime

Reply:
Let's do the next structure in onCreateView()


        val view = inflater.inflate(R.layout.fragment_reel_picker, container, false)
        applyArguments()
        applyAttributes()
        setFragmentResultListeners()
        initializeView(view)
        return view


!!!! Please make the same methods structure in DateFragment and TimeFragment

Reply:
![image](https://user-images.githubusercontent.com/79689735/184638293-beab9444-b144-4896-b7fd-5995c9cf54fd.png)

Reply:
![image](https://user-images.githubusercontent.com/79689735/184638360-2d19d1b8-b3af-45a8-b0e0-985d2cdf60b1.png)

Reply:
![image](https://user-images.githubusercontent.com/79689735/184638410-53bf0a84-abfd-4e6d-ae86-ef9652d9f66f.png)